src/window_skill.cpp

```diff
@@ -84,7 +84,7 @@ void Window_Skill::DrawItem(int index) {
 		bool enabled = CheckEnable(skill_id);
 		int color = !enabled ? Font::ColorDisabled : Font::ColorDefault;
 
-		contents->TextDraw(rect.x + rect.width - 24, rect.y, color, fmt::format("-{:3d}", costs));
+		contents->TextDraw(rect.x + rect.width - 24, rect.y, color, fmt::format("{:s}{:3d}", lcf::rpg::Terms::TermOrDefault(lcf::Data::terms.easyrpg_skill_cost_separator, "-"), costs));
```

Comment:
This is a string character, you can omit the ``{:s}`` is the default -> ``{}`` (same above)